tests/integration/test_mongodb.py

```diff
@@ -25,6 +25,9 @@
 @only_with_juju_secrets
 @pytest.mark.abort_on_fail
 async def test_deploy(ops_test: OpsTest, app_charm: PosixPath, data_integrator_charm: PosixPath):
+    if (await ops_test.model.get_status()).model.version.startswith("3.4."):
+        pytest.skip("Test is incompatible with Juju 3.4")
+
```

Comment:
VM test seems to be passing on 3.4

Reply:
@dragomirp - I think if it is passing then we should allow the tests to be ran on 3.4 ⭐ 

@Mehdi-Bendriss what do you think? :)

Reply:
+1, we hadn't tested mongodb on 3.4 - but since they're passing, it's good news and we can make mongodb follow as well :)

Reply:
@Mehdi-Bendriss does that mean we can allow the tests here to run on 3.4?

Reply:
IIRC VM passes and K8s didn't. I can check the cloud name here and just run VM. Should I update the PR?

Reply:
Enabled Mongo VM tests.

Reply:
@dragomirp thank you. Yes, for VM. K8s is not yet ready for this jump - should happen a bit later this month